react/src/lib/List/index.js

```diff
@@ -177,14 +177,14 @@ class List extends React.Component {
       case 37:
         this.getNextFocusedChild(items, tgt, -1);
         this._needsRefocus = true;
-        flag = true;
+        if(!shouldPropagateKeyDown) flag = true;
         break;
 
       case 39:
       case 40:
         this.getNextFocusedChild(items, tgt, 1);
         this._needsRefocus = true;
-        flag = true;
+        if(!shouldPropagateKeyDown) flag = true;
```

Comment:
This is backwards compatible as long as no overrides come in the future...I'm not certain this is the way to handle it but it's difficult to know until we have all other possible edge cases @andrewholsted

Reply:
Just want to mention: One of the challenges of including accessibility in the library.

Reply:
Yeah.. Most of Accessibility changes find their way into changes in momentum-ui code. 😃